ocean_provider/routes/compute.py

```diff
@@ -282,23 +282,23 @@ def computeStart():
     logger.info("Sending: %s", workflow)
 
     tx_id = data.get("transferTxId")
-    did = data.get("documentId")
     compute_env = data.get("environment")
     seconds = (
         datetime.fromtimestamp(validator.valid_until) - datetime.utcnow()
     ).seconds
 
     nonce, provider_signature = sign_for_compute(provider_wallet, consumer_address)
+    web3 = get_web3()
     payload = {
         "workflow": workflow,
         "providerSignature": provider_signature,
-        "documentId": did,
-        "agreementId": tx_id,
+        "agreementId": data.get("dataset").get("transferTxId"),
```

Comment:
dataset and transferTxId are required as far as I know, so you can do data["dataset"]["transferTxId"]